File: app/MyJobPitch-New/app/src/main/java/com/myjobpitch/api/data/Interview.java
package com.myjobpitch.api.data;

import com.myjobpitch.api.MJPAPIObject;

import java.util.Date;
import java.util.List;

/**
 * Created by Kei on 26/06/2018.
 */
public class Interview extends ApplicationInterview {
    private Integer application;
    private Date cancelled;
    private Integer cancelled_by;
    private String status;

    public Integer getApplication() {
        return application;
    }

    public Date getCancelled() {
        return cancelled;
    }

    public Integer getCancelled_by() {
        return cancelled_by;
    }

    public String getStatus() {
        return status;
    }
}

File: ios/MyJobPitch/Utils/API/models/Interview.swift
//
//  Interview.swift
//  MyJobPitch
//
//  Created by TIGER1 on 6/30/18.
//  Copyright © 2018 myjobpitch. All rights reserved.
//

import UIKit

class Interview: MJPObject {
    static let mappingArray = [ "id", "at", "notes", "feedback", "cancelled", "status" ]
    static let mappingDictionary = [ "cancelledBy": "cancelled_by" ]
    
    var at: Date!
    var messages: NSArray!
    var notes: String!
    var feedback: String!
    var cancelled: Date!
    var cancelledBy: NSNumber!
    var status: String!
}

class InterviewForSave: MJPObject {
    static let mappingArray = [ "invitation", "application", "at", "notes", "feedback" ]
    
    var invitation:String!
    var at: Date!
    var application: NSNumber!
    var notes: String!
    var feedback: String!
}
